doc/btp_spec.txt

```diff
@@ -3760,3 +3772,718 @@ Commands and responses:
+					UUID_Length (1 octet)
+					UUID (2 or 16 octets)
+
+		Valid UUID_Length parameter values:
+					0x02 = UUID16
+					0x10 = UUID128
+
+		If Status is non-zero Descriptors_Count shall be set to 0.
+
+	Opcode 0x87 - Read Characteristic Value/Descriptor Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+					Data_Length (2 octets)
+					Data (variable)
+
+		If Status is non-zero Data_Length shall be set to 0.
+
+	Opcode 0x88 - Read Using Characteristic UUID Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+					Data_Length (2 octets)
+					Value_Len (1 octet)
+					Data (variable)
+
+		Data is array of handle-value pairs defined as:
+					Handle (2 octets)
+					Value (Value_Len octets)
+
+		If Status is non-zero Data_Length shall be set to 0.
+
+	Opcode 0x89 - Read Long Characteristic Value/Descriptor Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+					Data_Length (2 octets)
+					Data (variable)
+
+		If Status is non-zero Data_Length shall be set to 0.
+
+	Opcode 0x8a - Read Multiple Characteristic Values Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+					Data_Length (2 octets)
+					Data (variable)
+
+		If Status is non-zero Data_Length shall be set to 0.
+
+	Opcode 0x8b - Write Characteristic Value/Descriptor Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+
+	Opcode 0x8c - Write Long Characteristic Value/Descriptor Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+
+	Opcode 0x8d - Reliable Write Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+
+	Opcode 0x8e - Configure Notifications Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+
+	Opcode 0x8f - Configure Indications Response
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Status (1 octet)
+
+	Opcode 0x90 - Notification/Indication Received
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Type (1 octet)
+					Handle (2 octets)
+					Data_Length (2 octets)
+					Data (variable)
+
+		Valid Type parameter values:
+					0x01 = Notification
+					0x02 = Indication
+					0x03 = Multiple Notification
+
+		This event indicates that IUT has received notification
+		or indication.
+
+	Opcode 0x91 - Multiple Handle Notification Received
```

Comment:
This might be not needed, because we don't have means to easily determine that notification comes from `ATT_MULTIPLE_HANDLE_VALUE_NTF` - IUTs (Both Zephyr and Mynewt) just handle these notifications one by one. What we can expect is multiple `0x90 `events

Reply:
it is IUT implementation detail but I guess we can add it later on if there is explicit need for this